### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by Mte90** [DanZaidan/break_arcade_games_out#2]
There is any plan to compile that also for linux?

**Comment by furai** [comment]
Yeah, wish it wasn't windows dependant. I wanted to have a look into it as I'm learning C now.

**Comment by ghost** [comment]
Changing the media backend and overall removing/replacing win32 would be plenty of work. 

This project is unmaintained but the permissive license helps alot, so it is possible, but not in this repo.

Some work on this project can be automated out with some good regexps and smart vim macros.

If you need to compile it look into cross-compiling, though I am weary if that is a surefire method.
 
Use large quantities of wine, if all you do want is to play, the scale is small enough to not have to worry about performance.
